config.json

```
@@ -715,8 +740,15 @@
         "slug": "hamming",
         "name": "Hamming",
         "uuid": "05162ee0-38ac-40ad-a591-d70a74b6963a",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "if-statements",
+          "for-loops"
```

Maybe also add `linq`?

```suggestion
          "for-loops",
          "linq"
```

---

I though about that one, but didn't know if it was allowed.